src/dropdown/Dropdown.js

```diff
@@ -131,7 +224,8 @@ export default class Dropdown extends React.Component {
       className: cx('dropdown', className, this.getCustomClassNames()),
       value: this.valueToOption(this.getValue(), options),
       onInputKeyDown,
-      onChange: _onChange
+      onChange: _onChange,
+      menuRenderer: _menuRenderer
```

Comment:
As already mentioned, you're completely ignoring the `menuRenderer` possibly coming from props. I'd set `_menuRenderer` as the default value for `menuRenderer` prop.